packit_service/worker/helpers/build/babysit.py

```diff
@@ -141,7 +141,7 @@ def check_copr_build(build_id: int) -> bool:
         bool: Whether the run was successful, False signals the need to retry.
     """
     logger.debug(f"Getting copr build ID {build_id} from DB.")
-    builds = CoprBuildTargetModel.get_all_by_build_id(build_id)
+    builds = list(CoprBuildTargetModel.get_all_by_build_id(build_id))
```

Comment:
It doesn't look like we need a list here, cause it calls `update_copr_builds` and is used there just in 2 `for-each` loops

Reply:
Yes, but we first want to check whether it's empty and to do that I'd either need to do
- `next(iter()))` or
- call `.first()`, which would make `mypy` angry because `error: "Iterable[CoprBuildTargetModel]" has no attribute "first"`
So I decided that this would be the easiest.